OpenTabletDriver.Plugin/Tablet/IDeviceReport.cs

```diff
@@ -1,7 +1,11 @@
+using System;
+
 namespace OpenTabletDriver.Plugin.Tablet
 {
     public interface IDeviceReport
     {
         byte[] Raw { get; }
+        string GetStringRaw() => BitConverter.ToString(Raw).Replace('-', ' ');
+        string GetStringFormat() => "";
```

Comment:
I like this change, however maybe we should just make it pull the raw report when its just an IDeviceReport.
```suggestion
        string GetStringFormat() => GetStringRaw();
```